api-common/src/main/java/io/enmasse/api/auth/KubeAuthApi.java

```diff
@@ -146,7 +190,29 @@ public TokenReview performTokenReview(String token) {
             resourceAttributes.put("verb", verb);
 
             spec.put("resourceAttributes", resourceAttributes);
-            spec.put("user", user);
+            spec.put("user", tokenReview.getUserName());
+            if (tokenReview.getUserId() != null && !tokenReview.getUserId().isEmpty()) {
```

Comment:
Did you mean to guard an empty string username?   I suspect, no, but if yes, then presumably you want to do the same thing for group names and extra names.